server/services/caseloadService.ts

```diff
@@ -199,7 +199,10 @@ export default class CaseloadService {
           licences: licences.map(licence => {
             return {
               id: licence.licenceId,
-              status: <LicenceStatus>licence.licenceStatus,
+              status:
+                licence.kind === 'HARD_STOP' && licence.isReviewNeeded
```

Comment:
shouldn't need this `kind` check - the point of adding the `isReviewNeeded` flag to every licence type was that this would work for immediate releases without changes